### PR TITLE
Creation of Wish table

### PR DESCRIPTION
**Pull request by Gmarn**
# Summary

- Add model in migrate;
- Configure the model.

# Trello Card

https://trello.com/c/1BJPxR5j

### CODE REVIEW

File: app/models/book_suggestion.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class BookSuggestion < ApplicationRecord
   has_one :user
+  has_many :wishs
```
Comment (francoprud) [inline app/models/book_suggestion.rb]: Change this to `wishes`

File: app/models/user.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ class User < ApplicationRecord
   validates :first_name, :last_name, presence: true
 
   has_many :rents
+  has_many :wishs
```
Comment (francoprud) [inline app/models/user.rb]: Change this to `wishes`

File: app/models/wish.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Wish < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :user_id, :book_id, presence: true
```
Comment (francoprud) [inline app/models/wish.rb]: Stop and think. What's the difference between validating the presence of `user_id` and validating the presence of `user`? (same for `book_id` and `book`)

Get the conclusion with examples

Comment (Gmarn) [inline app/models/wish.rb]: That nice, that exactly what i was looking for. At finals was simple. Thanks.

File: app/models/book_suggestion.rb (hunk 1 above)
Comment (Gmarn) [inline app/models/book_suggestion.rb]: That was in the wrong file :/

**Pull request merged**